<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.walletapplication.payme.service.AccountOpsTest" tests="10" skipped="0" failures="0" errors="0" timestamp="2024-01-16T10:40:12" hostname="U-nishi" time="0.096">
  <properties/>
  <testcase name="testDeleteAccount2()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.008"/>
  <testcase name="testDeleteAccount3()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.004"/>
  <testcase name="testGetAccount()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.004"/>
  <testcase name="testUpdateAccount()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.01"/>
  <testcase name="testGetAccount2()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.003"/>
  <testcase name="testGetAccount3()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.005"/>
  <testcase name="testDeleteAccount()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.004"/>
  <testcase name="testUpdateAccount2()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.007"/>
  <testcase name="testUpdateAccount3()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.005"/>
  <testcase name="testUpdateAccount4()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.025"/>
  <system-out><![CDATA[16:10:12.533 [Test worker] ERROR AccountOps -- Wallet does not exist: 42
16:10:12.539 [Test worker] INFO AccountOps -- Wallet retrieved successfully, wallet id 42
16:10:12.551 [Test worker] INFO AccountOps -- Wallet updated successfully, wallet id 42
16:10:12.557 [Test worker] ERROR AccountOps -- Wallet does not exist: 42
16:10:12.571 [Test worker] INFO AccountOps -- Wallet deleted successfully, wallet id 42
16:10:12.587 [Test worker] ERROR AccountOps -- Wallet does not exist: 42
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
